corehq/apps/export/export.py

```diff
@@ -336,19 +309,19 @@ def get_export_file(export_instances, filters, progress_tracker=None):
 def get_export_documents(export_instance, filters):
     # Pull doc ids from elasticsearch and stream to disk
     query = _get_export_query(export_instance, filters)
-    _, temp_path = tempfile.mkstemp()
-    with open(temp_path, 'w') as f:
-        scroll_result = query.scroll_ids()
-        for doc_id in scroll_result:
-            f.write(doc_id + '\n')
+    scroll_result = query.scroll_ids()
 
     def iter_export_docs():
-        # Stream doc ids from disk and fetch documents from ES in chunks
-        with open(temp_path) as f:
-            doc_ids = (doc_id.strip() for doc_id in f)
-            for doc in iter_es_docs(query.index, doc_ids):
-                yield doc
-        os.remove(temp_path)
+        with TransientTempfile() as temp_path:
+            with open(temp_path, 'w') as f:
```

Comment:
I think this is using the default `open` when it should be using `io.open` and passing in `encoding="utf-8"`. Same below on line 321.

Reply:
updated

Reply:
Just reverted that change because it caused build failures.  I'm just gonna leave these as they were beforehand and deal with making this code python 3 compatible in another PR.